Makefile

```diff
@@ -105,7 +106,6 @@ types:
 static-checks: lint lint-security types
 
 prepare-spacy:
-	poetry install -E spacy
```

Comment:
for context: I removed this, because `make install full` is called in the ci workflow beforehand, so spacy is already installed. Executing this here just leads to other extras being deinstalled.